patches/web3-core@1.8.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/types/index.d.ts b/types/index.d.ts
+index c3be2b5af3ca0d5dd6bbeb3815dbc08bbf60ef89..7de6e3da9d4615e328b2d0bd2f024f698794103b 100644
+--- a/types/index.d.ts
++++ b/types/index.d.ts
+@@ -217,7 +217,7 @@ export interface TransactionReceipt {
+     logs: Log[];
+     logsBloom: string;
+     events?: {
+-        [eventName: string]: EventLog;
```

Comment:
Did you open an issue on web3? If they accept this change, we can remove this patch in the future.